### PR TITLE
Add copier sequence diagram

### LINKED ISSUE #9: Create a workflow diagram of the user experience
The diagram will help clearly visualize the goal and purpose of the copier tool. It will also help orient development efforts to get this tool ready for users. 

UPDATE: Will add this diagram to the README to help provide background to people who want to contribute or maintain this repo.

### PR DESCRIPTION
**Pull request by bonicim**
Closes #17

### CODE REVIEW

File: copier_workflow.txt [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+User->Copier: Start Copier agent wizard (similar to vcfg experience)
+
+note over User,Copier: User answers questions to generate Agent source code from templates
+
+Copier->Agent files: Create agent directory and source files
+
+
+User->svn: Create agent remote repo
+
+User->svn: Push agent files to remote repo
```
Comment (kefeimo) [inline copier_workflow.txt:18]: Consider integrating Craig's comment in https://github.com/VOLTTRON/copier-poetry-volttron-agent/issues/9#issuecomment-1076901232

User->Copier: Start Copier agent wizard ("Runs copier command")

File: copier_workflow.txt [hunk 2]
```diff
@@ -0,0 +1,18 @@
+title Copier Workflow
+
+# participants
+actor User
+participant Copier
+participant Agent files
+participant "Github/Gitlab/Bitbucket/Etc" as svn
+
+User->Copier: Start Copier agent wizard (similar to vcfg experience)
```
Comment (kefeimo) [inline copier_workflow.txt:9]: I am not sure how much is "vcfg experience" relevant here.

File: copier_workflow.txt [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+note over User,Copier: User answers questions to generate Agent source code from templates
+
+Copier->Agent files: Create agent directory and source files
```
Comment (kefeimo) [inline copier_workflow.txt:13]: Is "Agent files" meaning "Copier agent template" (i.e., with jinja), or "Copier-generated files".

Discussion: unify the terminology. I have been using the terms: "Copier source code" (i.e., from which pip install copier), "Copier template"/"Agent template" (i.e., the one we are working on), "Copier generated files"/"Agent artifacts".

**Comment by kefeimo** [comment]
The following figure from [Copier's readthedocs](https://copier.readthedocs.io/en/stable/) indicates the workflow.
image

Note the term: template, project.

File: copier_workflow.txt (hunk 3 above)
Comment (kefeimo) [reply copier_workflow.txt:13]: The following figure from [Copier's readthedocs](https://copier.readthedocs.io/en/stable/) indicates the workflow.
image

Note the term: template, project.

**Review by kefeimo — CHANGES_REQUESTED** [review]

File: copier_workflow.txt (hunk 2 above)
Comment (bonicim) [reply copier_workflow.txt:9]: Relevant in the sense that 'vcfg' asks you several questions to help setup your Volttron platform. The copier tool also asks you several questions to help setup your Agent source code.

File: copier_workflow.txt (hunk 1 above)
Comment (bonicim) [reply copier_workflow.txt:18]: Will do.

File: copier_workflow.txt (hunk 3 above)
Comment (bonicim) [reply copier_workflow.txt:13]: Agent files means "Copier-generated files".

I don't understand what you mean by unifying the terminology. We should talk over the phone about what you mean.

**Pull request merged**